engine/test/integration_tests/e2e_node_failure/run.sh

```diff
@@ -1,21 +1,20 @@
 #!/bin/bash
 set -eu
 
-CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 WORK_DIR=$OUT_DIR/$TEST_NAME
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+
 CONFIG="$DOCKER_COMPOSE_DIR/3m3e.yaml"
+CONFIG=$(adjust_config $OUT_DIR $TEST_NAME $CONFIG)
```

Comment:
It's better to leave some comments on some special action.
Or It may be hard to by maintained  by other developer.

Reply:
We add some check logic in run.sh, ref: https://github.com/pingcap/tiflow/pull/6672/files#diff-bf84dcac2516965b250409c0ddb84cda64e4fb9c026c411744de9ed502d06bb9R38